_episodes/15-transferring-files.md

```diff
@@ -8,183 +8,107 @@ objectives:
 - "Be able to transfer files to and from a computing cluster."
 keypoints:
 - "`wget` downloads a file from the internet."
-- "`sftp`/`scp` transfer files to and from your computer."
+- "`scp` transfer files to and from your computer."
 - "You can use an SFTP client like FileZilla to transfer files through a GUI."
 ---
 
-One thing people very frequently struggle with is transferring files to and from a cluster. We'll
-cover several methods of doing this from the command line, then cover how to do this using the GUI
-program FileZilla, which is much more straightforwards.
+Computing with a remote computer offers very limited use if we cannot get files to 
+or from the cluster. There are several options for transferring data between computing 
+resources, from command line options to GUI programs, which we will cover here.
 
-## Grabbing files from the internet
+## Download files from the internet using wget
 
-To download files from the internet, the easiest tool to use is `wget`. The syntax is relatively
-straightforwards: `wget https://some/link/to/a/file.tar.gz` We've actually done this before to
-download our example files:
+One of the most straightforward ways to download files is to use `wget`. Any file 
+that can be downloaded in your web browser with an accessible link can be downloaded 
+using `wget`. This is a quick way to download datasets or source code. 
+
+The syntax is: `wget https://some/link/to/a/file.tar.gz`. For example, download the 
+lesson sample files using the following command:
 
 ```
 [remote]$ wget https://hpc-carpentry.github.io/hpc-intro/files/bash-lesson.tar.gz
 ```
 {: .bash}
 
-## Transferring single files and folders with scp
 
-To copy a single file to or from the cluster, we can use `scp`. The syntax can be a little complex
-for new users, but we'll break it down here:
+##  Compressing files
 
-To transfer *to* another computer:
-```
-[local]$ scp /path/to/local/file.txt yourUsername@remote.computer.address:/path/on/remote/computer
-```
-{: .bash}
+Let's look at the file we just downloaded:
 
-To download *from* another computer:
 ```
-[local]$ scp yourUsername@remote.computer.address:/path/on/remote/computer/file.txt /path/to/local/
+[remote]$ ls
+bash-lesson.tar.gz
 ```
-{: .bash}
```

Comment:
Please restore the `{: .bash}` tag for syntax highlighting,
and separate out the output into its own block with a `{: .output}` tag.